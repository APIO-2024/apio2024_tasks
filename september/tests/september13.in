txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 5
0 0 2 2 1 0
3 5 6 4 1 2
5 3 4 6 1 2
5 3 4 6 2 1
3 5 6 4 1 2
3 5 4 6 2 1
10 5
0 0 2 3 4 0 3 6 8
7 5 9 4 6 8 3 2 1
5 7 9 4 6 8 3 1 2
7 9 5 4 8 6 3 1 2
7 5 9 4 8 6 3 2 1
7 5 9 4 8 6 3 1 2
7 5
0 0 1 2 0 5
6 5 3 2 4 1
6 3 2 4 1 5
6 2 4 1 5 3
6 2 4 5 3 1
6 2 4 3 1 5
6 5
0 1 1 1 0
2 4 3 1 5
2 4 3 5 1
2 4 3 5 1
2 4 3 1 5
2 4 3 5 1
