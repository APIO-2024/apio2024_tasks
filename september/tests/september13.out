7ckgnn4wyi495puj3ibqf81dqvapyv6b
3
5
2
4
